R/html_document.R

```diff
@@ -689,4 +719,49 @@ navbar_link_text <- function(x, ...) {
     tagList(x$text, ...)
 }
 
+add_anchor_sections <- function(anchor_sections, section_divs = FALSE) {
+
+  res <- list(args = NULL, lua_filters = NULL, extra_dependencies = NULL)
 
+  if (xfun::isFALSE(anchor_sections)) return(res)
```

Comment:
You also check this above?

Reply:
Oh yes.... 🤔 must be a duplicate introduced when I refactored the code out of  `html_document`. Thanks!